test/test_community/test_community.py

```diff
@@ -0,0 +1,35 @@
+import unittest
+import requests_mock
+from parsons import Table, Community
+from unittest.mock import MagicMock
+
+
+TEST_CLIENT_ID = "someuuid"
+TEST_CLIENT_TOKEN = "somesecret"
+
+TEST_ID = "12345"
+TEST_URI = "https://faketestingurl.com/example"
+
+TEST_CSV_TYPE = "campaigns"
+
+TEST_GET_RESPONSE = {
```

Comment:
Can we rename this or leave a code comment that makes it clearer that this is mock data equivalent to the response from community.com that contains a csv string?